api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -248,4 +249,39 @@ package object models {
   }
 
   implicit lazy val MetronomeInfoWrites: Writes[MetronomeInfo] = Json.writes[MetronomeInfo]
+
+  implicit lazy val MarathonRunSpecWrites: Writes[RunSpec] = new Writes[RunSpec] {
+    override def writes(runSpec: RunSpec): JsValue = Json.obj(
+      "cpu" -> runSpec.cpus,
+      "mem" -> runSpec.mem,
+      "cmd" -> runSpec.cmd,
+      "container" -> runSpec.container.toString,
+      "args" -> runSpec.args,
+      "env" -> runSpec.env.toString(),
+      "constraints" -> runSpec.constraints.toString(),
+      "roles" -> runSpec.acceptedResourceRoles
+    )
+  }
+
+  implicit lazy val QueuedTaskInfoWrites: Writes[QueuedTaskInfo] = new Writes[QueuedTaskInfo] {
+    override def writes(taskInfo: QueuedTaskInfo): JsValue = Json.obj(
+      "runid" -> taskInfo.runSpec.id.toString,
+      "toLaunch" -> taskInfo.tasksLeftToLaunch,
+      "backoff" -> taskInfo.backOffUntil.toDateTime,
+      "runspec" -> taskInfo.runSpec
```

Comment:
runSpec?

Reply:
yeah...noted above... we should probably call this runs